### PR TITLE
Translate src/c & README.md to ENG

### PR DESCRIPTION
**Pull request by BDadmehr0**
There was no comment in most of the C codes and I added them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1691,7 +1691,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td><a href="https://pt.wikipedia.org/wiki/Deque_(estruturas_de_dados)">Deque</a></td>
+            <td><a href="https://pt.wikipedia.org/wiki/Deque_(estruturas_de_dados)">Deck</a></td>
```
Comment (kelvins) [inline README.md:1694]:
```suggestion
            <td><a href="https://pt.wikipedia.org/wiki/Deque_(estruturas_de_dados)">Double-ended Queue</a></td>
```

File: README.md [hunk 2]
```diff
@@ -1749,7 +1749,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td><a href="https://pt.wikipedia.org/wiki/FIFO">Fila</a></td>
+            <td><a href="https://pt.wikipedia.org/wiki/FIFO">Row</a></td>
```
Comment (kelvins) [inline README.md:1752]:
```suggestion
            <td><a href="https://pt.wikipedia.org/wiki/FIFO">Queue</a></td>
```

File: README.md [hunk 3]
```diff
@@ -1865,7 +1865,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td><a href="https://pt.wikipedia.org/wiki/Teoria_dos_grafos">Grafo</a></td>
+            <td><a href="https://pt.wikipedia.org/wiki/Teoria_dos_grafos">Fork</a></td>
```
Comment (kelvins) [inline README.md:1868]:
```suggestion
            <td><a href="https://pt.wikipedia.org/wiki/Teoria_dos_grafos">Graph</a></td>
```

File: README.md [hunk 4]
```diff
@@ -1923,7 +1923,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td><a href="https://www.youtube.com/watch?v=bxwIm3F6aaQ">Lista Circular Ligada</a></td>
+            <td><a href="https://www.youtube.com/watch?v=bxwIm3F6aaQ">Linked Circular List</a></td>
```
Comment (kelvins) [inline README.md:1926]:
```suggestion
            <td><a href="https://www.youtube.com/watch?v=bxwIm3F6aaQ">Circular Linked List</a></td>
```

File: README.md [hunk 5]
```diff
@@ -1807,7 +1807,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td><a href="https://www.ime.usp.br/~pf/algoritmos/aulas/lista.html">Fila Encadeada Dinâmica</a></td>
+            <td><a href="https://www.ime.usp.br/~pf/algoritmos/aulas/lista.html">Dynamic Chained Queue</a></td>
```
Comment (kelvins) [inline README.md:1810]:
```suggestion
            <td><a href="https://www.ime.usp.br/~pf/algoritmos/aulas/lista.html">Dynamic Queue</a></td>
```

File: README.md [hunk 6]
```diff
@@ -2213,7 +2213,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td><a href="https://pt.wikipedia.org/wiki/LIFO">Pilha</a></td>
+            <td><a href="https://pt.wikipedia.org/wiki/LIFO">Battery</a></td>
```
Comment (kelvins) [inline README.md:2216]:
```suggestion
            <td><a href="https://pt.wikipedia.org/wiki/LIFO">Stack</a></td>
```

File: README.md [hunk 7]
```diff
@@ -2155,7 +2155,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td>Lista Sequencial Ordenada</td>
+            <td>Ordered Sequential List</td>
```
Comment (kelvins) [inline README.md:2158]:
```suggestion
            <td>Ordered List</td>
```

File: README.md [hunk 8]
```diff
@@ -2271,7 +2271,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td>Pilha Ligada Dinâmica</td>
+            <td>Dynamic Connected Stack</td>
```
Comment (kelvins) [inline README.md:2274]:
```suggestion
            <td>Dynamic Stack</td>
```

File: README.md [hunk 9]
```diff
@@ -3493,7 +3493,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td><a href="https://pt.wikipedia.org/wiki/Pal%C3%ADndromo">Palíndromo</a></td>
+            <td><a href="https://pt.wikipedia.org/wiki/Pal%C3%ADndromo">Sum of 2 Numbers</a></td>
```
Comment (kelvins) [inline README.md:3496]:
```suggestion
            <td><a href="https://pt.wikipedia.org/wiki/Pal%C3%ADndromo">Palindrome</a></td>
```

File: README.md [hunk 10]
```diff
@@ -3551,7 +3551,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td><a href="https://pt.wikipedia.org/wiki/F%C3%B3rmula_de_Leibniz_para_%CF%80">Calculo do PI (Leibniz)</a></td>
+            <td><a href="https://pt.wikipedia.org/wiki/F%C3%B3rmula_de_Leibniz_para_%CF%80">Calculation of PI (Leibniz)</a></td>
```
Comment (kelvins) [inline README.md:3554]:
```suggestion
            <td><a href="https://pt.wikipedia.org/wiki/F%C3%B3rmula_de_Leibniz_para_%CF%80">Leibniz Formula for PI</a></td>
```

File: README.md [hunk 11]
```diff
@@ -3609,7 +3609,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
             </td>
         </tr>
         <tr>
-            <td>Busca em Labirinto</td>
+            <td>Labyrinth Search</td>
```
Comment (kelvins) [inline README.md:3612]:
```suggestion
            <td>Maze-solving Algorithm</td>
```

File: src/c/AlgoritmoDijkstra.c [hunk 12]
```diff
@@ -79,9 +79,9 @@ int main(){
 
     Dijkstra(Matriz, nroVertices);
 
-    printf("Total caminho mais curto do vertice 0 ao 4: %lu\n", Matriz[4][0]); // Caminho total mais curto
+    printf("Total caminho mais curto do vertice 0 ao 4: %lu\n", Matriz[4][0]); // Shortest total path
```
Comment (kelvins) [inline src/c/AlgoritmoDijkstra.c:82]:
```suggestion
    printf("Shortest path from vertex 0 to 4: %lu\n", Matriz[4][0]); // Shortest total path
```

**Review by kelvins — COMMENTED** [review]
Thanks @BDadmehr0, could you add the README changes to a separate PR?

Also, it's being very painful to review this PR due to its size, it would be nice to create separate PRs for groups of algorithms (e.g. 5 algorithms per PR), it would be much easier to review and faster to merge.

**Comment by BDadmehr0** [comment]
[kelvins](https://github.com/kelvins) I will definitely provide the README changes in another RP

**Comment by kelvins** [comment]
Closing stale PR (more than 30 days without interactions).
Feel free to reopen it if you still want to contribute. 🍻

**Pull request closed without merging**